delivery_postlogistics/tests/test_packaging_code.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo.exceptions import UserError
+from odoo.tests import Form, TransactionCase
```

Comment:
Is using `TestPostlogisticsCommon` relevant here too? 
```suggestion
from odoo.tests import Form
from .common import TestPostlogisticsCommon
```

If not, fallback on `BaseCommon`